### PR TITLE
Prepare for release v1.7.0

### PR DESCRIPTION
**Pull request by torredil**
Signed-off-by: Eddie Torres <torredil@amazon.com>

### CODE REVIEW

**Comment by torredil** [comment]
cc: @wmesard

**Comment by torredil** [comment]
Sorry for the noise. 
/retest-required

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ jobs:
             ## CHANGELOG
             See [CHANGELOG](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/CHANGELOG-0.x.md) for full list of changes
           draft: false
-          prerelease: false
+          prerelease: true
```
Comment (olemarkus) [inline .github/workflows/release.yaml:26]: Curious about this change. It seems unusual to flag a semver stable tag as a prerelease.

Comment (olemarkus) [reply .github/workflows/release.yaml:26]: I think the quick fix of this would be to have the job set `draft: true` rather than `prerelease: true`.

**Pull request merged**